bin/pyrdp-player.py

```diff
@@ -63,19 +72,35 @@ def main():
     parser.add_argument("-p", "--port", help="Bind port (default: 3000)", default=3000)
     parser.add_argument("-o", "--output", help="Output folder", default="pyrdp_output")
     parser.add_argument("-L", "--log-level", help="Log level", default="INFO", choices=["INFO", "DEBUG", "WARNING", "ERROR", "CRITICAL"], nargs="?")
+    parser.add_argument("--headless", help="Parse a replay without rendering the user interface.", action="store_true")
 
     args = parser.parse_args()
     outDir = Path(args.output)
     outDir.mkdir(exist_ok = True)
 
     logLevel = getattr(logging, args.log_level)
-    prepareLoggers(logLevel, outDir)
+    prepareLoggers(logLevel, outDir, args.headless)
 
-    app = QApplication(sys.argv)
-    mainWindow = MainWindow(args.bind, int(args.port), args.replay)
-    mainWindow.show()
+    if not HAS_GUI and not args.headless:
+        logging.error('Headless mode is not specified and PySide2 is not installed. Install PySide2 to use the graphical user interface.')
+        exit(127)
```

Comment:
The `exit` method exists in this context? Shouldn't this be `sys.exit()`?

Reply:
It does, but I think it's an interpreter built-in, so it might not exist on all implementations of Python. I'll change it to `sys.exit`